docs/shipwrightbuild.md

```diff
@@ -17,9 +17,9 @@ When the `ShipwrightBuild` instance is created, the following components are ins
   - `buildpacks-v3`
   - `buildpacks-v3-heroku`
   - `kaniko`
-  - `kaniko-trivy`
   - `ko`
   - `source-to-image`
+  - `source-to-image-redhat`
```

Comment:
Reviewer note - as a matter of policy, the operator will not delete `ClusterBuildStrategy` objects if they are removed in a release. Doing so risks broken builds that depend on a previously released build strategy.